x-pack/plugins/security_solution/server/endpoint/mocks.ts

```diff
@@ -6,6 +6,8 @@
 
 import { ILegacyScopedClusterClient, SavedObjectsClientContract } from 'kibana/server';
 import { loggingSystemMock, savedObjectsServiceMock } from 'src/core/server/mocks';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
```

Comment:
We may want to ask the Kibana Platform team to include this in `src/core/server/mocks`

Reply:
@paul-tavares I see you approved, thank you! But let's follow up and figure out a better way to do this. +++